packages/nc-gui/components/project/spreadsheet/components/extras.vue

```diff
@@ -29,65 +30,86 @@
           :class="{ active: true }"
           dense
         >
-          <v-list-item dense href="https://discord.gg/5RgZmkW" target="_blank">
-            <!-- Join Discord -->
+          <v-list-item>
+            <div class="d-flex justify-space-between d-100 pr-2">
+              <v-icon v-t="['community:discord']" class="mr-1" size="22" :color="textColors[0]" @click="open('https://discord.gg/5RgZmkW','_blank')">
```

Comment:
I think the second parameter is not necessary as the default target in `open()` is `_blank` already.

Reply:
ya true, i will update